src/inputs/datePickerInput/datePickerInput.jsx

```diff
@@ -19,6 +19,12 @@ import DatePickerCalendarOnClickOutside from '../datePickerCalendar/datePickerCa
 import withStyles from '../../styles/withStyles';
 
 const propTypes = {
+    /**
+     * Forces the Input component to always show the required indicator
```

Comment:
I thought about that but went with Input instead since it's literally the component being rendered with the required flag, but I'll change the comment since it does seem more consistent that way.

Reply:
Looks good, and thanks.  Resolving.